tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -3539,3 +3539,928 @@ Feature: an user gets the resources shared to them
+                      "driveId",
+                      "driveType"
+                    ],
+                    "properties": {
+                      "driveId": {
+                        "type": "string",
+                        "pattern": "%space_id_pattern%"
+                      },
+                      "driveType" : {
+                        "type": "string",
+                        "enum": ["project"]
+                      }
+                    }
+                  },
+                  "permissions": {
+                    "type": "array",
+                    "maxItems": 1,
+                    "minItems": 1,
+                    "items": {
+                      "type": "object",
+                      "required": [
+                        "grantedToV2",
+                        "id",
+                        "invitation",
+                        "roles"
+                      ],
+                      "properties": {
+                        "id": {
+                          "type": "string",
+                          "pattern": "^%permissions_id_pattern%$"
+                        },
+                        "grantedToV2": {
+                          "type": "object",
+                          "required": ["user"],
+                          "properties": {
+                            "user": {
+                              "type": "object",
+                              "properties": {
+                                "displayName": {
+                                  "type": "string",
+                                  "enum": ["Brian Murphy"]
+                                },
+                                "id": {
+                                  "type": "string",
+                                  "pattern":"^%user_id_pattern%$"
+                                }
+                              },
+                              "required": [
+                                "displayName",
+                                "id"
+                              ]
+                            }
+                          }
+                        },
+                        "invitation": {
+                          "type": "object",
+                          "required": ["invitedBy"],
+                          "properties": {
+                            "invitedBy": {
+                              "type": "object",
+                              "required": ["user"],
+                              "properties": {
+                                "user": {
+                                  "type": "object",
+                                  "required": [
+                                    "displayName",
+                                    "id"
+                                  ],
+                                  "properties": {
+                                    "displayName": {
+                                      "type": "string",
+                                      "enum": ["Alice Hansen"]
```

Comment:
```suggestion
                                      "enum": [""]
```
Since the user `Alice` is deleted but Alice is displayed. I think 2-3 second needed for user deletion.